web/frontend/src/components/slam/index.svelte

```diff
@@ -93,6 +101,22 @@ const startDurationTimer = (start: number) => {
   }, 400);
 };
 
+const setMappingMode = async () => {
+  mappingMode = overrides?.mappingDetails.mode ?? 'undefined';
+  if (
+    overrides !== undefined &&
+    overrides.slamModel !== '' &&
+    overrides.slamModel !== 'viam:slam:cartographer'
```

Comment:
Fine with leaving these in for now, but we should make a ticket for after this project to remove the special casing for cartographer where we can. Cartographer returns the mapping mode in its properties already so we should be able to use those here.

EDIT: since we are adding the slamModel in this pr, maybe we can just remove the special casing?

Reply:
Ah, yes, I was looking at how we set the mapping mode for Cartographer in app: https://github.com/viamrobotics/app/blob/578d47d40f6f58dd5629b95fd152ed2875acb970/frontend/src/components/tabs/control.vue#L176

And didn't think through if that's still needed given that we just call `Properties`. I can remove and simplify that part, then!

Reply:
Fixed it: https://github.com/viamrobotics/app/pull/4194/files#diff-1625b9dcfbcd13c9358071c47bb18e97e117b7f510324e14e9329c023c772275R178-R187